e2e/tests/truffle/test/Event.js

```diff
@@ -0,0 +1,99 @@
+const Web3 = require('web3')
+const fs = require('fs')
+const path = require('path')
+
+const {
+    SpeculativeNonceTxMiddleware,
+    SignedTxMiddleware,
+    Client,
+    LocalAddress,
+    CryptoUtils,
+    LoomProvider
+} = require('loom-js')
+
+const EventTestContract = artifacts.require('EventTestContract')
+
+contract('EventTestContract', async (accounts) => {
+    let contract, from, nodeAddr
+
+    beforeEach(async () => {
+        nodeAddr = fs.readFileSync(path.join(process.env.CLUSTER_DIR, '0', 'node_rpc_addr'), 'utf-8').trim()
+        const chainID = 'default'
+        const writeUrl = `ws://${nodeAddr}/websocket`
+        const readUrl = `ws://${nodeAddr}/queryws`
```

Comment:
We also need to test all this with the `/eth` endpoint

Reply:
yes done